homeassistant/components/panasonic_viera/media_player.py

```diff
@@ -103,12 +100,12 @@ def available(self) -> bool:
         return self._remote.available
 
     @property
-    def volume_level(self):
+    def volume_level(self) -> int | None:
```

Comment:
```suggestion
    def volume_level(self) -> float | None:
```
The volume level is between 0 and 1.